### PR TITLE
Update prerequisite-docker.md

### PR DESCRIPTION
**Pull request by arthuo**
Translate the English part of the document

### CODE REVIEW

File: site/zh-CN/getstarted/prerequisite-docker.md [hunk 1]
```diff
@@ -4,34 +4,34 @@ label: 使用 Docker Compose 安装
...
+
+在安装 Milvus 之前，请检查你的硬件和软件是否满足要求。
 
 {{tab}}
 
-## Hardware requirements
+## 硬件要求
 
-| Component           | Requirement                                                  |Recommendation| Note                                                         |
+|硬件           | 前提                                                  |建议配置| 注意                                                         |
```
Comment (LocoRichard) [inline site/zh-CN/getstarted/prerequisite-docker.md:17]: 前提--》要求
注意--》说明

File: site/zh-CN/getstarted/prerequisite-docker.md [hunk 2]
```diff
@@ -4,34 +4,34 @@ label: 使用 Docker Compose 安装
...
-| CPU instruction set | <ul><li>SSE4.2</li><li>AVX</li><li>AVX2</li><li>AVX-512</li></ul> |<ul><li>SSE4.2</li><li>AVX</li><li>AVX2</li><li>AVX-512</li></ul> |  Vector similarity search and index building within Milvus require CPU's support of single instruction, multiple data (SIMD) extension sets. Ensure that the CPU supports at least one of the SIMD extensions listed. See [CPUs with AVX](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions#CPUs_with_AVX) for more information.                           |
-| RAM                 | <ul><li>standalone: 16G</li><li>cluster: 64G</li></ul>       |<ul><li>standalone: 32G</li><li>cluster: 128G</li></ul>        | The size of RAM depends on the data volume.                  |
-| Hard drive          | SATA 3.0 SSD or higher                                       |SATA 3.0 SSD or higher | The size of hard drive depends on the data volume.           |
+| CPU                 | Intel CPU Sandy Bridge 或以上                              |<ul><li>单机版： 8 core 或更多</li><li>分布式版： 16 core 或更多</li></ul>| 当前版本的 Milvus 不支持 AMD 和 Apple M1 CPU。 |
```
Comment (LocoRichard) [inline site/zh-CN/getstarted/prerequisite-docker.md:19]: core--》核

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by LocoRichard** [comment]
Hi @arthuo , I have made necessary feedbacks, and they apply to the other PR, too. 

I hope that you have received the monthly updates of Milvus Community. We have announced you as our Contributor of the month. Congratulations, and have a nice day!

**Comment by arthuo** [comment]
> Hi @arthuo , I have made necessary feedbacks, and they apply to the other PR, too.
> 
> I hope that you have received the monthly updates of Milvus Community. We have announced you as our Contributor of the month. Congratulations, and have a nice day!

thank you for your feedback. At the same time, I am excited by the news.

**Comment by LocoRichard** [comment]
> > Hi @arthuo , I have made necessary feedbacks, and they apply to the other PR, too.
> > I hope that you have received the monthly updates of Milvus Community. We have announced you as our Contributor of the month. Congratulations, and have a nice day!
> 
> thank you for your feedback. At the same time, I am excited by the news.

It's probably because you are not in the mail list if you did not receive the email.
Here is the [Email](https://mailchi.mp/63fe4d9a4227/milvus-monthly-updates-for-february-2022?e=d98d989a7e).
You can subscribe to it if you do not want to miss anything.

Merging the PRs.

**Pull request merged**